test/test_torch.py

```diff
@@ -377,7 +383,7 @@ def test_addmm(self):
                 for j in range(25):
                     for k in range(50):
                         res2[i, j] += m1[i, k] * m2[k, j]
-            self.assertEqual(res1, res2)
+            self.assertEqual(res1, res2, _prec)
```

Comment:
Usually underscore notation means unused variable, so you might want to rename it to `prec`.